src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php

```diff
@@ -48,24 +49,54 @@ public function vote(TokenInterface $token, $subject, array $attributes)
             }
 
             // as soon as at least one attribute is supported, default is to deny access
-            $vote = self::ACCESS_DENIED;
+            $vote = $this->deny();
 
-            if ($this->voteOnAttribute($attribute, $subject, $token)) {
+            $decision = $this->voteOnAttribute($attribute, $subject, $token);
+            if (\is_bool($decision)) {
+                trigger_deprecation('symfony/security-core', '5.3', 'Returning a boolean in "%s::voteOnAttribute()" is deprecated, return an instance of "%s" instead.', static::class, Vote::class);
```

Comment:
```suggestion
                trigger_deprecation('symfony/security-core', '5.4', 'Returning a boolean in "%s::voteOnAttribute()" is deprecated, return an instance of "%s" instead.', static::class, Vote::class);
```

Reply:
in other files as well

Reply:
Sorry, too many files. I fixed all that I can find now. Please let me know if I miss anything.